tests/parsing/errors/other/expected/hashIdent.res.txt

```diff
@@ -0,0 +1,37 @@
+
+  Syntax error!
+  tests/parsing/errors/other/hashIdent.res:1:10-12
+
+  1 │ let x = #10s
+  2 │ 
+  3 │ type t = [ #red | #10s ] 
+
+  A numeric polymorphic variant cannot be followed by a letter. Did you mean `#10`?
```

Comment:
@bloodyowl what do you think of this new message? Specialized it to give a clear suggestion on what the user could do in this case.

Reply:
Yup, that looks better 👍